### PR TITLE
chore: added PR template

### PR DESCRIPTION
**Pull request by 10ko**
Introduces a PR template. 
I just took [the one from the kubernetes project](https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUEST_TEMPLATE.md) (thanks L) and adapted it to our needs (mostly removed stuff since our processes are a bit simpler for now).

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, please read our contributor guidelines in the CONTRIBUTING.md file.
```
Comment (ellenkorbes) [inline .github/PULL_REQUEST_TEMPLATE.md]: We can make it easier on people by providing the URL directly. (https://github.com/garden-io/garden/blob/master/CONTRIBUTING.md)

Comment (10ko) [reply .github/PULL_REQUEST_TEMPLATE.md]: Right! I'll fix it, the file originally was in the root and when I moved it to the `.github/` folder I didn't think of updating the link.

**Comment by eysi09** [comment]
We should rebase and merge this.

**Comment by 10ko** [comment]
I'll rebase.

**Pull request merged**